packages/utils/src/time.ts

```diff
@@ -37,89 +17,56 @@ interface Performance {
   now(): number;
 }
 
+/**
+ * Returns a timestamp in seconds since the UNIX epoch using the Date API.
+ *
+ * TODO(v8): Return type should be rounded.
+ */
+export function dateTimestampInSeconds(): number {
+  return Date.now() / ONE_SECOND_IN_MS;
+}
+
 /**
  * Returns a wrapper around the native Performance API browser implementation, or undefined for browsers that do not
  * support the API.
  *
  * Wrapping the native API works around differences in behavior from different browsers.
  */
-function getBrowserPerformance(): Performance | undefined {
-  const { performance } = WINDOW;
+function unixTimestampInSecondsFunc(): () => number {
+  const { performance } = GLOBAL_OBJ as typeof GLOBAL_OBJ & { performance: Performance };
```

Comment:
```suggestion
  const { performance } = GLOBAL_OBJ as typeof GLOBAL_OBJ & { performance?: Performance };
```

maybe a bit more accurate here?

Reply:
Done 395fd8e6b289ab05957475fe12d7722e94773692